packages/typegpu/src/core/sampler/sampler.ts

```diff
@@ -197,6 +199,20 @@ class TgpuFixedSamplerImpl implements TgpuFixedSampler, SelfResolvable {
       _props.mipmapFilter === 'linear';
   }
 
+  /**
+   * NOTE: Internal use only
+   */
+  unwrap(branch: ExperimentalTgpuRoot): GPUSampler {
```

Comment:
Use `Unwrapper` type instead of `ExperimentalTgpuRoot`
```suggestion
  unwrap(branch: ExperimentalTgpuRoot): GPUSampler {
```